strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/dependency/snippet/PypiWheelDependencyFormatterTest.java

```diff
@@ -0,0 +1,130 @@
+package org.carlspring.strongbox.dependency.snippet;
+
+import org.carlspring.strongbox.artifact.coordinates.PypiWheelArtifactCoordinates;
+import org.carlspring.strongbox.dependency.snippet.PypiWheelDependencyFormatter;
+import org.carlspring.strongbox.providers.layout.PypiLayoutProvider;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import javax.inject.Inject;
+import java.util.*;
+
+/**
+*  Class to test the functionality of PypiWheelDependencyFormatter
+*/
+public class PypiWheelDependencyFormatterTest
+{
+  private ArrayList<String> wheelartifactcoordinates = new ArrayList<String>();
+  private ArrayList<String> formattedwheels = new ArrayList<String>();
+  @Inject
+  private CompatibleDependencyFormatRegistry compatibleDependencyFormatRegistry;
+  
+  public PypiWheelDependencyFormatterTest()
+  {
+        PypiWheelDependencyFormatter formatter = new PypiWheelDependencyFormatter();
+        assertNotNull(formatter, "Failed to look up dependency synonym formatter!");
```

Comment:
**Cosmetic:** Could you please put an empty line before and after assertion blocks? It helps them stand out and makes them much more easier to spot when going through the code. Thanks! :)